src/aws-cpp-sdk-core/source/utils/logging/CRTLogging.cpp

```diff
@@ -37,84 +33,53 @@ static int s_aws_logger_redirect_log(
         const char *format, ...)
 {
     AWS_UNREFERENCED_PARAM(logger);
-    Aws::Utils::Threading::ReaderLockGuard guard(CRTLogSystemLock);
-    if (CRTLogSystem)
-    {
-        assert(logger->p_impl == &s_sdkCrtLogger);
-        Aws::Utils::Logging::LogLevel logLevel = static_cast<LogLevel>(log_level);
-        const char* subjectName = aws_log_subject_name(subject);
-        va_list args;
-        va_start(args, format);
-        CRTLogSystem->Log(logLevel, subjectName, format, args);
-        va_end(args);
-        return AWS_OP_SUCCESS;
-    }
-    else if (s_CRTLogsRedirectionIsSet)
+    if (!CRTLogSystem)
     {
-        // SDK CRT logger is terminated, fallback to an original/previous CRT logger
```

Comment:
I just thought about it more and realized that we don't control / own "original" logger, so better to not try to over-smart ourselves.